Targets/V8/Patches/v8.patch

```diff
@@ -0,0 +1,188 @@
+diff --git a/BUILD.bazel b/BUILD.bazel
```

Comment:
If you want, feel free to send a CL to V8 to upstream these changes. Otherwise I'll do it after we land this PR.